PowerEditor/src/NppNotification.cpp

```diff
@@ -852,7 +852,7 @@ BOOL Notepad_plus::notify(SCNotification *notification)
 			// replacement for obsolete custom SCN_SCROLLED
 			if (notification->updated & SC_UPDATE_V_SCROLL)
 			{
-				addHotSpot();
+				addHotSpot(notifyView);

```

Comment:
I guess it's for fixing #9056.
Next time please do it separately, since this fix has nothing to do with the URL test issue.

Reply:
Sorry for that, it wasn't meant to be here. I just tested it quickly, but I forgot to remove it after the test.

Reply:
No problem. Just a friendly reminder.